notebooks/tutorials/1_getting_started_lesson4/notebook.ipynb

```diff
@@ -0,0 +1,406 @@
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Equity(209 [AM])</th>\n",
+       "      <td>0.001801</td>\n",
+       "      <td>-0.022995</td>\n",
+       "      <td>-0.038365</td>\n",
+       "      <td>2.370000</td>\n",
+       "      <td>2</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Equity(337 [AMAT])</th>\n",
+       "      <td>-0.002525</td>\n",
+       "      <td>-0.014339</td>\n",
+       "      <td>0.007575</td>\n",
+       "      <td>2.370000</td>\n",
+       "      <td>2</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                                     1         5        10  \\\n",
+       "date                      asset                                              \n",
+       "2013-01-02 00:00:00+00:00 Equity(52 [ABM])    0.004430  0.004430  0.004430   \n",
+       "                          Equity(114 [ADBE]) -0.015389  0.008086 -0.012259   \n",
+       "                          Equity(166 [AES])  -0.006368 -0.008104 -0.005403   \n",
+       "                          Equity(209 [AM])    0.001801 -0.022995 -0.038365   \n",
+       "                          Equity(337 [AMAT]) -0.002525 -0.014339  0.007575   \n",
+       "\n",
+       "                                                factor  factor_quantile  \n",
+       "date                      asset                                          \n",
+       "2013-01-02 00:00:00+00:00 Equity(52 [ABM])    2.560000                2  \n",
+       "                          Equity(114 [ADBE]) -1.896667                1  \n",
+       "                          Equity(166 [AES])  -2.630000                1  \n",
+       "                          Equity(209 [AM])    2.370000                2  \n",
+       "                          Equity(337 [AMAT])  2.370000                2  "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Import Alphalens\n",
+    "import alphalens as al\n",
+    "\n",
+    "# Get asset forward returns and quantile classification\n",
+    "# based on sentiment scores\n",
+    "factor_data = al.utils.get_clean_factor_and_forward_returns(\n",
+    "    factor=pipeline_output['sentiment_score'],\n",
+    "    prices=asset_prices,\n",
+    "    quantiles=2,\n",
+    "    periods=(1,5,10),\n",
+    ")\n",
+    "\n",
+    "# Display first 5 rows\n",
+    "factor_data.head(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Having our data in this format allows us to use several of Alphalens's analysis and plotting tools. Let's start by looking at the mean returns by quantile over the entire period. Because our goal is to build a long-short strategy, we want to see one quantile have negative returns and the other quantile have positive returns:"
```

Comment:
`we want to see one quantile have negative returns and the other quantile have positive returns:` -> `we want to see the lower quantile (1) have negative returns and the upper quantile(2) have positive returns:`